apps/demos/Demos/Chat/AIIntagration/jQuery/styles.css

```diff
@@ -0,0 +1,42 @@
+.demo-container {
+  display: flex;
+  justify-content: center;
+  padding: 20px;
```

Comment:
Demo frame already have padding 20px. I assume this one is excess

Reply:
[0f60119da81def3f3f4e8a295eaefda87674892c]